optuna/pruners/hyperband.py

```diff
@@ -50,6 +51,10 @@ class HyperbandPruner(BasePruner):
         at least :math:`4 \\times 10` pruners are consumed for startup.
 
     Args:
+        max_resource:
+            A parameter for specifying the maximum resource allocated to a trial noted as :math:`R`
+            in the paper. This value is exactly same with the maximum iteration steps (e.g.,
+            `max_epoch` for neural networks).
```

Comment:
Please use double backquote to annotate variables (e.g.., `step` and `EPOCH`in [SuccessiveHalvingPruner](https://optuna.readthedocs.io/en/stable/reference/pruners.html#optuna.pruners.SuccessiveHalvingPruner) reference).
```suggestion
            ``max_epoch`` for neural networks).
```